src/Builder/ReferenceEntityAttribute/ProductReferenceEntityAttributeValueValueBuilder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synolia\SyliusAkeneoPlugin\Builder\ReferenceEntityAttribute;
+
+final class ProductReferenceEntityAttributeValueValueBuilder
```

Comment:
Interface is missing.

Reply:
this one has no interface, the other interface is used to add builders to this class.